### PR TITLE
net-im/signal-desktop-bin: add new version 5.28.0

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

**Review by ionenwks — APPROVED** [review]

File: net-im/signal-desktop-bin/Manifest [hunk 1]
```diff
@@ -1,3 +1,2 @@
 DIST signal-desktop_5.26.1_amd64.deb 127893190 BLAKE2B 0d7dedf5370c1010d204ad0e13e1ca46f886eef4e2e51af1fd6f3cf163c7d6bc464d21719018dbc168818b696550fbc709a1ce6dbc242125ea856d62b2a24a4f SHA512 dffd694e637528587b4bcfa6303e41129e2872022f9e97b338ad71b4a763fedc52650d5b94a5cd1282119163968df599cf3b8bb7639879dfc4132fdcedfe7b2f
-DIST signal-desktop_5.27.1_amd64.deb 125206928 BLAKE2B 67f39b1c52564f8768307b56ef3f879e2130a3cb2aa1f650c8cfee3bf6d26f1ece6b4267fc3911802bc23c2c91875e1aff0d9f5d4cdb7e5f02d1a022063e35fd SHA512 1e81e2bb7c94b5866f7fcb93800c606c88696df5fb06791c52071dad1b7c9df2d7a2936a006c55d3d7a1b9155d4681212a6b41123447f2d76120ecbbeaba51a3
```
[the remark this answers is not in the document]
Comment (robert7k) [reply net-im/signal-desktop-bin/Manifest:2]: Alright, sounds like a plan. I'll file a stabilization bug in a few days for 5.28.0 then.

[the remark this answers is not in the document]
Comment (juippis) [reply net-im/signal-desktop-bin/Manifest:2]: I think it's maybe best if we just pushed this straight-to-stable every time.

[the remark this answers is not in the document]
Comment (ionenwks) [reply net-im/signal-desktop-bin/Manifest:2]: Yeah, I also think it'd be fine here.

These are pretty uneventful, maybe if there was some major changes but usually not the case.

[the remark this answers is not in the document]
Comment (robert7k) [reply net-im/signal-desktop-bin/Manifest:2]: Alright, will do that in the future in case nothing else has changed in the ebuild.

**Pull request closed without merging**